src/perf/telemetry-node.js

```diff
@@ -514,18 +514,25 @@ Telemetry.getEvolution = function Telemetry_getEvolution(channel, version,
       if (!Array.isArray(filter)) {
         filter = [filter];
       }
-
       for (let i = 0; i < filter.length; ++i) {
         filterString += `&${encodeURIComponent(filterName)}=${
           encodeURIComponent(filter[i])}`;
       }
     });
-  const url = `${Telemetry.BASE_URL}aggregates_by/${useSubmissionDate ?
+
+    if (metric === 'TIME_TO_NON_BLANK_PAINT_NO_NETOPT_MS' && version < '59') {
```

Comment:
Could you add a comment/link explaining what this is about for future reference?

Reply:
are we intentionally comparing to the string '59'?

Reply:
@spasovski Yes, because the existing code requires a string (line 52 of this file). @wlach sure, will do.